index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
+    <p>
+    From a graph model perspective, even though the nodes identified as the
+    <a>DID controller</a> and <a>DID subject</a> in figure 4 are distinct,
+    there is a logical arc connecting them to express a semantic equivalence
+    relationship (in RDF/OWL, this is expressed using the <a href="https://www.w3.org/TR/owl-ref/#sameAs-def">owl:sameAs predicate)</a>.
+    </p>
+    <h2>Set 2: The DID subject is <em>not</em> the DID controller</h2>
+    <p>
+    The second case, shown in figure 2 in the previous appendix, is when the
+    <a>DID subject</a> is a separate entity from the <a>DID controller</a>. This
+    is the case when, for example, a parent creates a <a>DID</a> for a child; a
+    corporation creates a <a>DID</a> for a subsidiary; or a manufacturer creates
+    a <a>DID</a> for a product, an IoT device, or a digital file.
+    </p>
+    <p>
+    From a graph model perspective, the only difference between figure 2 and
+    figure 4 is that in the latter there is an equivalence arc relationship
+    between the <a>DID subject</a> and <a>DID controller</a> nodes.
+    </p>
+  </section>
+
+<section class="appendix">
+    <h1>
+Multiple DID Controllers
+    </h1>
+    <p>
+    A <a>DID document</a> may have more than one <a>DID controller</a>. In this
+    situation there are two basic options available for how control is shared.
+    </p>
+    <h2>Option #1: Independent Control</h2>
+    <p>
+    In the first option, shown in figure 5, each of the <a>DID controllers</a>
+    may act on its own, i.e., each one has full power to update the
+    <a>DID document</a> independently. From a graph model perspective, in this
+    configuration:
+    </p>
+    <ul>
+        <li>
+        Each additional <a>DID controller</a> is another distinct graph node
+        (which may be identified by its own <a>DID</a>).
+        </li>
+        <li>
+        The same arcs (“controls” and “controller”) exist between each
+        <a>DID controller</a> and the <a>DID document</a>.
+        </li>
+    </ul>
+        <figure id="independent-did-controllers">
+          <img style="margin: auto; display: block; width: 75%;"
+            src="diagrams/figure-c.1-independent-did-controllers.png" alt="
+            Diagram showing three DID controllers each with an independent
+            control relationship with the DID document
+          " >
+          <figcaption>
+              Multiple independent <a>DID controllers</a> that can each act
+              independently
+          </figcaption>
+        </figure>
+
+    <h2>Option #2: Group Control</h2>
+    <p>
+    In the second option, the <a>DID controllers</a> must act together in some
+    fashion, such as when using a cryptographic algorithm that requires multiple
+    digital signatures (“multi-sig”) or a threshold number of digital
+    signatures (“m-of-n”). From a functional standpoint, this option is similar
+    to a single <a>DID controller</a> because, although each of the
```

Comment:
Maybe it is worth, for laypersons, to have some sort of a reference or some example somewhere else in the document how a `multi-sig` or `m-of-n` scheme can be added to a DID document. This is the only case in the appendix where the DID references are not immediately clear based on the core spec itself...

Reply:
For multi-sig we could include multiple Link Data Security proofs as an array in a document showing the old state and the new state being signed by multiple keys from the old state.

If we want to show a simple threshold option, we could talk about using a Shamir's secret sharing scheme, but from an example perspective there's not a great solution for this because this feature is actually managed at the key management layer. Another example that would be more complicated, but could show good examples would be to use a BLS Threshold signature. However, I don't know of anyone that's got running code to show that working properly and able to produce legitimate signatures with. If we just want some example signatures that don't actually verify we could use them from [this CFRG draft](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-04), but we'd have no additional documentation about the LD security suite that would be associated with it because no one has actually created and properly documented that work.

Reply:
@iherman and @kdenhartog I agree with both of you that such examples would be helpful for the spec. However I don't think this appendix is the place for them vs. someplace in the main body of the spec. If one or more examples are added there, then we can add links to them from here.